.circleci/config.yml

```diff
@@ -56,9 +56,11 @@ aliases:
     run:
       name: bundle install
       command: |
+        sudo gem update --system
```

Comment:
I used this to troubleshoot an issue and to ensure we had up to date environment. Slows down the build and could be risky. Could make part of a different PR.

Reply:
Doesn't seem like it slows things down _too_ much so probably fine to keep in 😇